sanic_openapi/swagger.py

```diff
@@ -14,14 +14,7 @@
 dir_path = os.path.dirname(os.path.realpath(__file__))
 dir_path = os.path.abspath(dir_path + '/ui')
 
-
-# Redirect "/swagger" to "/swagger/"
-@blueprint.route('', strict_slashes=True)
-def index(request):
-    return redirect("{}/".format(blueprint.url_prefix))
```

Comment:
This redirect should be reserved because it can make the endpoint `/swagger` work until there is any better solution.